Gopkg.toml

```diff
@@ -20,46 +20,32 @@
 #  name = "github.com/x/y"
 #  version = "2.4.0"
 
+# TODO: update to 1.1.0
 [[constraint]]
   name = "github.com/container-storage-interface/spec"
   version = "1.0.0"
 
+# TODO: update to latest
 [[constraint]]
   name = "github.com/kubernetes-csi/csi-test"
   version = "~v1.0.3"
 
+# TODO: remove when official 3.0.0 tagged
 [[constraint]]
-  name = "github.com/kubernetes-csi/external-snapshotter"
-  version = "1.0.0-rc4"
-
-[[constraint]]
-  name = "github.com/kubernetes-sigs/sig-storage-lib-external-provisioner"
-  version = "v2.2.0"
+  name = "sigs.k8s.io/sig-storage-lib-external-provisioner"
+  version = ">=v3.0.0-beta"
 
+# TODO: can this be removed?
 [[constraint]]
   name = "google.golang.org/grpc"
   version = "1.9.2"
 
-[[constraint]]
-  name = "k8s.io/apimachinery"
-  version = "kubernetes-1.13.0-beta.1"
-
+# TODO: remove when official 11.0.0 tagged
```

Comment:
official *1.14.0* ?

Reply:
Client-go has a separate versioning scheme than kubernetes. 11.0 corresponds to 1.14

Reply:
ack